service/src/main/java/uk/nhs/adaptors/gp2gp/common/constants/Constants.java

```diff
@@ -0,0 +1,6 @@
+package uk.nhs.adaptors.gp2gp.common.constants;
+
+public class Constants {
+    public static final String DOCUMENT_TASK = "DocumentTask";
```

Comment:
The task names need to be more specific. The name should also include an action / verb. What is a task if it doesn't DO something?

GetGpcDocumentTask, GetGpcStructuredTask